imap_processing/__init__.py

```diff
@@ -8,3 +8,29 @@
 
 # Eg. imap_module_directory = /usr/local/lib/python3.11/site-packages/imap_processing
 imap_module_directory = Path(__file__).parent
+
+instruments = [
+    "codice",
+    "glows",
+    "hi",
+    "hit",
+    "idex",
+    "lo",
+    "mag",
+    "swapi",
+    "swe",
+    "ultra",
+]
+
+processing_levels = {
+    "codice": ["l0", "l1a", "l1b", "l2"],
+    "glows": ["l0", "l1a", "l1b", "l2"],
+    "hi": ["l0", "l1a", "l1b", "l1c", "l2"],
+    "hit": ["l0", "l1a", "l1b", "l2"],
+    "idex": ["l0", "l1a", "l1b", "l2"],
+    "lo": ["l0", "l1a", "l1b", "l1c", "l2"],
+    "mag": ["l0", "l1a", "l1b", "l1c", "l2"],
+    "swapi": ["l0", "l1", "l2"],
+    "swe": ["l0", "l1a", "l1b", "l2"],
+    "ultra": ["l0", "l1a", "l1b", "l2"],
```

Comment:
I still need to confirm some of these

Reply:
Should these be in each instrument's base module information? So ultra's information would stick with ultra and they could update it themselves?

Reply:
IDEX will only have L1 and L2